SR paperwork templates

ErhardSteinhauer opened the pull request:
## About the PR
- Added 8 paper templates for SR with forms for: employment contract, employee's pay history, fund requisition form, authorization contract, shuttle title change form, derelict ship form, contraband turnover form, contraband permit form (in case no sheriff on duty).
- Mapped SR folder with said templates to SR's office.

## Why / Balance
QoL

## How to test
1. Spawn as SR, look for green folder in the office, inspect papers inside that folder.

## Media
![2024-7-24_11 26 04](https://github.com/user-attachments/assets/37791136-7c0b-49be-91fc-c9760752cf17)
![2024-7-24_11 26 57](https://github.com/user-attachments/assets/f8471524-d989-412c-ade7-51f1ee0a0a84)
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- add: Added a folder with paperwork templates to SR's office.

dvir001 commented:
Based

whatston3 commented on Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml:
```diff
@@ -0,0 +1,272 @@
+# Forms
+- type: entity # Based on form made by Willzile1 (Jon-Gah)
+  id: PaperWrittenFrontierFormEmployment
+  name: employment contract
```
Thoughts on capitalizing these names to match the title?  I know it's just a form, but it's still titled.

ErhardSteinhauer replied:
None. If you think it'll be better that way, then sure.

whatston3 reviewed (changes requested):
Removed a bunch of redundancy (do you really need to write out the name of the SR in the form when their signature is at the bottom) and fixed some inconsistent tone.

I've put my rework up on my repo, changes can be seen here: https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-SR-Paperwork-Forms...whatston3:2024-07-24-SR-Paperwork-Forms-suggestions?expand=1

Not entirely happy with the changes, but language should be reasonably consistent between the forms.  Will also need to give space law a quick perusal before signing off on this, but it's looking good.

ErhardSteinhauer commented on Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml:
```diff
@@ -0,0 +1,272 @@
+
+        [color=#AAAAAA]This Form is void without signs of both Parties and a stamp of Frontier Command (Sheriff or Station Representative)[/color]
+
+- type: entity # Based on form made by Blumbee (Aki Suboshi)
+  id: PaperWrittenFrontierFormContrabandPermit
+  name: contraband permit form
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+        [color=#2e9935ff]◥[bold]N[/bold]◣ [bold]Frontier Outpost Command[/bold][/color]
+        [color=#2e9935ff]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+        [head=2]Tier 2 Contraband Permit[/head]
+
+
+        [bold]Permit Holder:[/bold] 
+        [bold]Holder Crew:
+        [bullet/] 
+        [bullet/] 
+        [bullet/] 
+
+        [bold]Sheriff on Duty:[/bold] 
+        [bold]Time of issuing:[/bold] 
+
+
+        This Permit allows the Holder and his crew,[italic] as written above[/italic], to hold and use C2 Contraband.
+        Non-compliance with Contraband Turnover orders void this Permit.
+        [bold]Class 2 Contraband contains:[/bold] [color=red]all explosive devices not otherwise covered by higher classifications; objects which can be used to non-destructively or non-invasively gain unauthorized access to Nanotrasen secured areas; exclusively restricted and/or authorized use weapons or equipment, uniforms and EMPs; and, security vessels.[/color]
+
+
+
+        [color=#AAAAAA]This Form is void without signs of both Parties and a stamp of NFSD acting Sheriff[/color]
```
I think SR can issue contraband permits if there is no sheriff around. Might be wrong though.

ErhardSteinhauer commented:
> Removed a bunch of redundancy (do you really need to write out the name of the SR in the form when their signature is at the bottom) and fixed some inconsistent tone.
> 
> I've put my rework up on my repo, changes can be seen here: https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-SR-Paperwork-Forms...whatston3:2024-07-24-SR-Paperwork-Forms-suggestions?expand=1
> 
> Not entirely happy with the changes, but language should be reasonably consistent between the forms. Will also need to give space law a quick perusal before signing off on this, but it's looking good.
> 
> Edit: also removed references to "acting" in the valid clause, because forms may still be considered valid by future sheriffs or station reps.

Good stuff, you should merge it. Actually, is there a way for me to merge it?

whatston3 commented:
> Good stuff, you should merge it. Actually, is there a way for me to merge it?

If you have my repo as a remote, you can merge it into your branch.  Assuming you have it checked out and don't have my repo as a remote:
```
git remote add whatstone https://github.com/whatston3/frontier-station-14.git
git merge whatstone/2024-07-24-SR-Paperwork-Forms-suggestions
```

ErhardSteinhauer commented:
> > Good stuff, you should merge it. Actually, is there a way for me to merge it?
> 
> If you have my repo as a remote, you can merge it into your branch. Assuming you have this branch checked out and don't have my repo as a remote:
> 
> ```
> git remote add whatstone https://github.com/whatston3/frontier-station-14.git
> git merge whatstone/2024-07-24-SR-Paperwork-Forms-suggestions
> ```
> 
> The branch name can be copied nicely from GitHub URLs, which is convenient.

I'm getting `not something we can merge` upon attempt to execute that command
